### DISCUSSION TITLE
Qwen for Chinese Market

### DISCUSSION THREAD

**Discussion by LvDAO** [langflow-ai/langflow#5396] [Ideas]
Qwen2.5 was released recently, which has been ranked 1st in open-source models, but it seems that langflow has no Qwen support currently. 
Why not do a Qwen implementation ^_^?

**Comment by ogabrielluiz** [comment]
Hey @LvDAO ! 

We support Ollama and LMStudio. I'm using Qwen almost everyday on my experiments in Langflow.

**Reply by LvDAO** [reply]
@ogabrielluiz Yeah, thank you for replying, Ollama is great for local deploy, what about the official Qwen API provided by AliCloud ?
